pom.xml

```diff
@@ -220,7 +220,7 @@
           <plugin>
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-javadoc-plugin</artifactId>
-          </plugin>
+	  </plugin>
```

Comment:
my belief is that this is a tab, and tabs are bad.

But of course, there's no reason to think anyone could get this right if we don't checkstyle the pom.xml. Boo.

(Underlying issue: checkstyle is in `build-tools` and maven doesn't like cyclic build dependencies).